lib/authz/permissions.go

```diff
@@ -876,6 +876,17 @@ func definitionForBuiltinRole(clusterName string, recConfig types.SessionRecordi
 					},
 				},
 			})
+	case types.RoleMDM:
+		return services.RoleFromSpec(
+			role.String(),
+			types.RoleSpecV6{
+				Allow: types.RoleConditions{
+					Namespaces: []string{types.Wildcard},
+					Rules: []types.Rule{
+						types.NewRule(types.KindDevice, services.RW()),
```

Comment:
My gut is that you'll need more than this to support things like authorization/other Teleport boilerplate, but it was very much something I discovered as I went for the Okta service.

Reply:
From my initial testing this seems to be all that I need. I was a bit surprised too, but in truth MDM services are rather simple - a little more than a glorified gRPC client.